content/sensu-core/0.29/reference/ssl.md

```diff
@@ -140,6 +140,8 @@ sudo service rabbitmq-server stop{{< /highlight >}}
 ].
 {{< /highlight >}}
 
+_NOTE: If using multiple CA's, it may be necessary to include the `{depth, 2},` parameter under `ssl_options`. For more information about this attribute, see the [RabbitMQ SSL Reference documentation][rmq-ssl-depth]._
```

Comment:
```suggestion
_NOTE: If using multiple CAs, it may be necessary to include the `{depth, 2},` parameter under `ssl_options`. For more information about this attribute, see the [RabbitMQ SSL Reference documentation][rmq-ssl-depth]._
```